testsuite/MDAnalysisTests/core/test_atomgroup.py

```diff
@@ -1058,6 +1064,12 @@ def test_UnWrapFlag_residues(self, ag, ref_Unwrap_residues):
         assert_almost_equal(ag.moment_of_inertia(unwrap=True, compound='residues'), ref_Unwrap_residues['MOI'], self.prec)
         assert_almost_equal(ag.asphericity(unwrap=True, compound='residues'), ref_Unwrap_residues['Asph'], self.prec)
 
+    def test_UnWrapFlag_residues_unordered(self, unordered_ag, ref_Unwrap_residues):
```

Comment:
This could be parametrized so each AG method is tested separately.

Reply:
You suggest also parameterizing the method? Like this?
```python
@pytest.mark.parametrize('method_name', ('center_of_geometry', 'center_of_mass',
                                         'moment_of_inertia', 'asphericity'))
def test_UnWrapFlag_residues_unordered(self, unordered_ag, ref_Unwrap_residues, method_name):
    method = getattr(unordered_ag, method_name)
    # requires that the keys of ref_Unwrap_residues are changed to match the method names
    assert_almost_equal(method(unwrap=True, compound='residues'),
                        ref_Unwrap_residues[method_name], self.prec)
```
I'm all for it, but several more tests are begging for the same cleanup. It seems a targeted refactoring effort would make more sense. If you think that at least new tests should be parameterized as above, I'm cool with having it thus.